packages/@uppy/locales/src/de_DE.js

```diff
@@ -149,6 +150,7 @@ de_DE.strings = {
   streamPassive: 'Stream passiv',
   submitRecordedFile: 'Aufgezeichnete Datei verwenden',
   takePicture: 'Ein Foto machen',
+  takePictureBtn: 'Foto aufnehmen',
```

Comment:
I think we should align the wording in `takePicture` and `takePictureBtn`. Maybe use 
'Ein Foto aufnehmen' and 'Foto aufnehmen'?